typedoc.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "cardstack",
+  "$schema": "https://typedoc.org/schema.json",
+  "out": "docs",
+  "entryPointStrategy": "packages",
+  "readme": "./README.md",
+  "includeVersion": true,
+  "plugin": ["typedoc-plugin-resolve-crossmodule-references"],
+  "exclude": [
+    "./packages/hub",
+    "./packages/cardpay-cli",
+    "./packages/boxel",
+    "./packages/eslint-config",
+    "./packages/demo-cards",
+    "./packages/firebase-functions",
+    "./packages/firebase-functions/review-feature/functions",
+    "./packages/firebase-functions/push/functions",
+    "./packages/firebase-functions/maintenance-status/functions",
+    "./packages/firebase-functions/minimum-version/functions",
+    "./packages/firebase-functions/check-version/functions",
+    "./packages/web-client",
+    "./packages/ember-shared/addon",
+    "./packages/ember-shared/test-app",
+    "./packages/compiled",
+    "./packages/ssr-web",
+    "./packages/safe-tools-subgraph",
+    "./packages/test-support",
+    "./packages/safe-tools-client",
+    "./packages/core",
+    "./packages/cardhost",
+    "./packages/base-cards",
+    "./packages/cardpay-subgraph",
+    "./scripts"
```

Comment:
All of this has been excluded for now. If we want to push stuff in feel free to uncomment.